build.fsx

```diff
@@ -88,14 +88,14 @@ Target "ConventionTests" (fun _ ->
     !! (sprintf "./Octokit.Tests.Conventions/bin/%s/**/Octokit.Tests.Conventions.dll" buildMode)
     |> xUnit2 (fun p -> 
             {p with
-                OutputDir = testResultsDir })
+                HtmlOutputPath = Some (testResultsDir @@ "xunit.html") })
```

Comment:
I dunno. I thought maybe AppVeyor or someone might've picked it up. Just thought I'd ask :smile: